examples/otlp-exporter-node/package.json

```diff
@@ -29,16 +29,16 @@
   },
   "dependencies": {
     "@opentelemetry/api": "^1.0.2",
-    "@opentelemetry/api-metrics": "0.29.1",
+    "@opentelemetry/api-metrics": "0.29.2",
     "@opentelemetry/core": "1.1.1",
-    "@opentelemetry/exporter-metrics-otlp-grpc": "0.29.1",
-    "@opentelemetry/exporter-metrics-otlp-http": "0.29.1",
-    "@opentelemetry/exporter-metrics-otlp-proto": "0.29.1",
-    "@opentelemetry/exporter-trace-otlp-grpc": "0.29.1",
-    "@opentelemetry/exporter-trace-otlp-http": "0.29.1",
-    "@opentelemetry/exporter-trace-otlp-proto": "0.29.1",
+    "@opentelemetry/exporter-metrics-otlp-grpc": "0.29.2",
```

Comment:
I guess you don't want to update the example.
Either don't do it or update also the stable entries to 1.3.1

Reply:
I'll just update. This is the only example in the monorepo for some reason but it isn't typescript so it isn't typechecked anyway. Looks like @pichlermarc added it to the monorepo. Was that intentional?